docs/management-roles.md

```diff
@@ -0,0 +1,18 @@
+# twinsphere Roles
+
+Here is a list of all available roles and their permissions, that you can set through Management API role assignment APIs.
+
+<!-- markdownlint-disable line-length -->
+| Role | Scope | Explanation |
+|------|-------|-------------|
+| organization-owner | organization | Full access rights across the organization |
+| tenant-administrator | cloud tenant | Full access rights for a given tenant |
+| tenant-global-writer | cloud tenant | Read and write permissions on all endpoints |
+| tenant-global-reader | cloud tenant | Read permissions on all endpoints |
+| tenant-metadata-writer | cloud tenant | Read and write permissions on all registry, discovery and concept descriptions endpoints (AAS metadata) |
+| tenant-metadata-reader | cloud tenant | Read permissions on all registry, discovery and concept descriptions endpoints (AAS metadata) |
+| tenant-push-service-user | cloud tenant | Permissions for all push service feature endpoints |
```

Comment:
we have separated roles here:
- "tenant-push-service-writer"
- "tenant-push-service-reader"